experimental/packages/opentelemetry-exporter-metrics-otlp-http/src/platform/browser/OTLPMetricExporter.ts

```diff
@@ -58,11 +55,10 @@ class OTLPExporterBrowserProxy extends OTLPExporterBrowserBase<ResourceMetrics,
 
   convert(
     metrics: ResourceMetrics[]
-  ): otlpTypes.opentelemetryProto.collector.metrics.v1.ExportMetricsServiceRequest {
-    return toOTLPExportMetricServiceRequest(
+  ): IExportMetricsServiceRequest {
+    return createExportMetricsServiceRequest(
```

Comment:
Same comment about `metrics[0]` here

Reply:
Changed in [d32f440](https://github.com/open-telemetry/opentelemetry-js/pull/2929/commits/d32f440b78f2ad6b3b21211b51e1c479eeeb520b) :slightly_smiling_face: